plugins/src/main/java/org/smartregister/fhir/gateway/plugins/LocationHierarchyEndpointHelper.java

```diff
@@ -161,7 +161,7 @@ public Bundle handleNonIdentifierRequest(
         List<String> selectedSyncLocations = extractSyncLocations(syncLocationsParam);
         String practitionerId = verifiedJwt.getSubject();
         List<String> userRoles = JwtUtils.getUserRolesFromJWT(verifiedJwt);
-        String syncStrategy = getSyncStrategy(verifiedJwt);
+        String syncStrategy = getSyncStrategyFromJwtToken(verifiedJwt);
```

Comment:
Nit pick (maybe) - I think we should have the method pass in the application id as parameter since conceptually we usually map an application id to a sync id.

Then call it something like like `getSyncStrategyByAppId(`